Source/WebCore/platform/graphics/transforms/TransformationMatrix.h

```diff
@@ -315,6 +316,10 @@ class TransformationMatrix {
     WEBCORE_EXPORT bool isInvertible() const;
     WEBCORE_EXPORT std::optional<TransformationMatrix> inverse() const;
 
+    // Rounds the x/y translation components to the nearest pixel (for non-perspective)
+    // transforms, and returns the subpixel offset that was removed.
+    LayoutSize round2dTranslationComponents(float deviceScaleFactor);
```

Comment:
I don't think LayoutUnits should ever show up in this header; this rounding should move elsewhere.